sql/moz-fx-data-shared-prod/udf_js/gunzip/udf.sql

```diff
@@ -70,6 +75,7 @@ unzipped AS (
 )
   --
 SELECT
-  mozfun.assert.equals(expected, result)
+  mozfun.assert.equals(expected, result),
+  mozfun.assert.equals(udf_js.gunzip(CAST(REPEAT("abc", 200000) AS BYTES)), REPEAT("abc", 200000)),
```

Comment:
Can this test case be written the same way as the two other above?

Reply:
Thanks for pointing that out.  I wasn't paying attention to the other tests